jwst/stpipe/utilities.py

```diff
@@ -29,10 +29,29 @@
 """
 Utilities
 """
+from copy import copy
+from importlib import import_module
 import inspect
+import logging
 import os
+from os import path
```

Comment:
File contains sys.path, os.path and path calls - might clean this up for readability (suggest os.path?).